src/engine/rarity_simulation.rs

```diff
@@ -852,8 +853,13 @@ impl Executor {
                         inputs: self.concrete_inputs.clone(),
                         pc: self.state.pc,
                     },
-                    instruction,
-                    operands: vec![],
+                    reason: UninitializedMemoryAccessReason::ReadUnintializedMemoryAddress {
+                        description: format!(
+                            "access to uninitialized memory at in write syscall ({:#x})",
```

Comment:
nit: s/at in/in/

Reply:
I think I see the problem. But I am not sure, how to interpret your command. Can you explain your command?  😄

Reply:
done

Reply:
Sorry, the `sed` substitute syntax took over my brain at some point. :smile:

https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html